plugin/trino-scylladb/pom.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>367-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-scylladb</artifactId>
+    <description>Trino - ScyllaDB Connector</description>
+    <packaging>trino-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencyManagement>
+        <!-- Create shaded ScyllaDB driver? -->
```

Comment:
add a `TODO` to avoid leaving this comment trailing behind

Reply:
I will add a TODO if we decide not to create a shaded library.

Reply:
This comment should be resolved before merge, so I won't add a TODO at this timing.